ale_linters/rust/cargo.vim

```diff
@@ -11,10 +13,18 @@ function! ale_linters#rust#cargo#GetCargoExecutable(bufnr) abort
     endif
 endfunction
 
+function! ale_linters#rust#cargo#GetCommand(buffer) abort
+    let l:command = g:ale_rust_cargo_use_check
+    \   ? 'check'
```

Comment:
Just wanted to note I hate this indentation style. Since it's Vimscript, Vim's default autoindent behavior should be considered the defacto official style...